src/views/Hero/index.view.tsx

```diff
@@ -27,7 +31,11 @@ const HeroView: React.FC<HeroProps> = ({
         <div className="HeroView__textContainer">
           <h1 className="HeroView__titleText">CruzHacks 2021 /</h1>
           <h1 className="HeroView__titleText--secondary">{title}</h1>
-          <p className="HeroView__description">{description}</p>
+          {description.map((item) => (
+            <p key={item.description} className="HeroView__description">
+              {item.description}
+            </p>
+          ))}
```

Comment:
I added this just to get rid of the array. I remember last year you pointed something out like this to me as well -- in order to get rid of static code and keeping it DRY, we can make use of these objects and create a mapping!